### PR TITLE
Feature/prod 169 correct number

### PR DESCRIPTION
**Pull request by imaji**

### CODE REVIEW

File: PusherServer.Tests/UnitTests/PusherOptions.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public void the_new_port_should_be_used_to_create_the_base_url_when_its_encrypte
             StringAssert.IsMatch("https://api.pusherapp.com:100", options.GetBaseUrl().AbsoluteUri);
         }
 
-        [Test]
+                [Test]
```
Comment (ewmy) [inline PusherServer.Tests/UnitTests/PusherOptions.cs]: Indent not required

File: PusherServer/Pusher.cs [hunk 2]
```diff
@@ -207,10 +197,11 @@ private TriggerBody CreateTriggerBody(string[] channelNames, string eventName, o
             ValidationHelper.ValidateChannelNames(channelNames);
             ValidationHelper.ValidateSocketId(options.SocketId);
 
+            // todo jmc this needs replacing
```
Comment (ewmy) [inline PusherServer/Pusher.cs]: hanging todo - is this done?

Comment (imaji) [inline PusherServer/Pusher.cs]: Yup, accidentally left the todo in

File: PusherServer.Tests/UnitTests/PusherOptions.cs (hunk 1 above)
Comment (imaji) [inline PusherServer.Tests/UnitTests/PusherOptions.cs]: Removed in coming commit

**Pull request merged**